worlds/witness/hints.py

```diff
@@ -226,6 +232,10 @@ def make_hints(world: MultiWorld, player: int, hint_amount: int):
 
     for item in always_items:
         hint_pair = make_hint_from_item(world, player, item)
+
+        if hint_pair[2] == 158007:
```

Comment:
this and the following might be worth a comment what this integer represents. I certainly don't get it right now.

Reply:
Right. It's Tutorial Gate Open, the location you have to do before you can even access a single audio log :D I'll add that